pkg/scheduler/framework/runtime/framework.go

```diff
@@ -1148,7 +1149,7 @@ func (f *frameworkImpl) RunScorePlugins(ctx context.Context, state *framework.Cy
 					logger := klog.LoggerWithName(logger, pl.Name())
 					ctx = klog.NewContext(ctx, logger)
 				}
-				s, status := f.runScorePlugin(ctx, pl, state, pod, nodeName)
+				s, status := f.runScorePlugin(ctx, pl, state, pod, nodeInfo)
```

Comment:
If some score plugins modify the `nodeInfo` in its implementation (e.g. modify node name). It would affect other plugins when they retrieve the original info nodeInfo, right?

Reply:
@haosdent Indeed, this would render the `nodeInfo` mutable, similar to the behavior observed with the `pod`.

Reply:
Got it